### ISSUE TITLE
Folded region sound

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#144927]
Based on feedback here https://github.com/microsoft/vscode/issues/142016

Let's go back to the old sound. I think the feedback is solid. The sound is not distinguishable with screen reader turned on.
If @gino-scarpino has time to find some middle ground solution that would also work

The sounds being too harsh might be less of a problem now since we have settings to disable per sound.

**Comment by hediet** [comment]
You updated the sound, right? Can we close this?

**Comment by isidorn** [comment]
@hediet yeah, let's close.

Feedback on the sound is very welcome. It is in today's insiders. Please try it out @steverep @cannona

**Comment by cannona** [comment]
It's definitely better. My only remaining concern is that most of the volume for this sound appears to be in the low-end. I'm not sure how audible it will be on a crappy laptop speaker, for example, that might have little to no low-end. I'll try it on my work laptop later today.

**Comment by isidorn** [comment]
@cannona thanks for the feedback. Let us know how it goes on the work laptop.

**Comment by steverep** [comment]
@isidorn  I agree it's definitely much better, but I also think the original sound was more distinguishable when overlaid with speech, probably because it contained higher frequencies and/or it was longer.

This might be a tougher feature to accomplish, but you might also look into a way to have the audio cues play before the screen reader starts reading the line of code, rather than overlaid on top of it.  If that were the case, then you would have more freedom in sound choices.  Until then, while the sounds have to compete with speech, they should be designed such that:

1. Default loudness meets or exceeds the screen reader speech, and
2. Sounds contain frequencies mostly outside the typical range for speech

And of course it goes without saying that, in general, the shorter the sound, the more likely it can blend in with speech and be missed.

**Comment by isidorn** [comment]
@steverep thank you for trying it out and providing feedback.

Unfortunately we can not control when the screen reader starts reading the text, so these two sounds are independent. So I do agree with you that we need to make the sounds that follow your 2. point.
For your 1. point we will make the audio cue volume configurable. This is coming later this week https://github.com/microsoft/vscode/issues/144771
